ledfx/api/effects.py

```
@@ -48,9 +51,163 @@ async def put(self, request: web.Request) -> web.Response:
+                        )
+                        effect_config_update[key] = not current_value
+                    else:
+                        effect_config_update[key] = value
+
+                # Apply the update if there are any valid keys
+                if effect_config_update:
+                    try:
+                        eff.update_config(effect_config_update)
+                        virtual.update_effect_config(eff)
+                        updated += 1
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)